compose_files/docker-compose-boot-node.yml

```diff
@@ -81,6 +81,8 @@ services:
       - nginx-proxy
     environment:
       - "DEFAULT_EMAIL=${ACME_DEFAULT_EMAIL?err_unset}"
+      - "ACME_CHALLENGE=${ACME_CHALLENGE_TYPE?err_unset}"
```

Comment:
I think since this variable is conditional it makes sense to set it to empty by default if not provided

```
"ACME_CHALLENGE=${ACME_CHALLENGE_TYPE:-}"
```